docs/advanced-usage.md

```diff
@@ -106,7 +106,25 @@ jobs:
 
 ## Nightly versions
 
-You can specify a nightly version to download it from https://nodejs.org/download/nightly.
+You can specify a nightly version to download it from https://nodejs.org/download/nightly. 
+
+### Install nightly build for specific node version
+
+```yaml
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    name: Node sample
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: '16.0.0-nightly' # it will install the latest nigthly release for node 16.0.0
+      - run: npm ci
+      - run: npm test
+```
+
+### Install nightly build for major node version
```

Comment:
```suggestion
### Install the nightly build for a major Node version
```

Reply:
Also, just for a better flow, I would move this first so it goes major version -> specific version -> exact nightly